app/assets/javascripts/_practice_partner_practices_utilities.es6

```diff
@@ -0,0 +1,9 @@
+(($) => {
+    const $document = $(document);
+
+    function execPageBuilderFunctions() {
+        replaceImagePlaceholders();
+    }
+
+    $document.on('turbolinks:load', execPageBuilderFunctions);
+})(window.jQuery);
```

Comment:
Creating this file was the only way I could figure out how to make this work. Calling the function in the existing `show.js.erb` file didn't work at all, I played around with calling it in the exec functions of `practice_card_utilities` but that resulted in the function being called 10x.. This ended up being the best working solution I could come up with.